classes/class-wc-connect-shipping-label.php

```diff
@@ -224,42 +242,42 @@ protected function get_form_schema() {
 				'properties' => array(
 					'height' => array(
 						'type' => 'number',
-						'title' => 'Height',
+						'title' => __( 'Height', 'woocommerce' ),
 					),
 					'product_id' => array(
 						'type' => 'string',
-						'title' => 'Product ID',
+						'title' => __( 'Product ID', 'woocommerce' ),
 					),
 					'length' => array(
 						'type' => 'number',
-						'title' => 'Length',
+						'title' => __( 'Length', 'woocommerce' ),
 					),
 					'quantity' => array(
 						'type' => 'number',
-						'title' => 'Quantity',
+						'title' => __( 'Quantity', 'woocommerce' ),
 					),
 					'weight' => array(
 						'type' => 'number',
-						'title' => 'Weight',
+						'title' => __( 'Weight', 'woocommerce' ),
 					),
 					'width' => array(
 						'type' => 'number',
-						'title' => 'Width',
+						'title' => __( 'Width', 'woocommerce' ),
 					),
 				),
 			);
 
 			$properties[ 'cart' ] = array(
 				'type' => 'array',
-				'title' => 'Items to send',
+				'title' => __( 'Items to send', 'woocommerce' ),
```

Comment:
Don't forget, packages !== items, and the label dialog doesn't really need to know about the concept of the "cart" - it really needs to know about the packages array (or singular package for this intiial revision.)